src/backends/compose/compose_backend.py

```diff
@@ -345,6 +345,9 @@ def default_config_args(self, tank):
         defaults += f" -rpcport={tank.rpc_port}"
         defaults += f" -zmqpubrawblock=tcp://0.0.0.0:{tank.zmqblockport}"
         defaults += f" -zmqpubrawtx=tcp://0.0.0.0:{tank.zmqtxport}"
+        # connect to initial peers as defined in graph file
+        for dst_index in tank.init_peers:
```

Comment:
We could print a warning here if we exceed n==8 so that someone would understand why not all their graph connections are being made (and in the k8s equivalent)

Reply:
Could that be added to the graph schema validate RPC?